CONTRIBUTING.md

```diff
@@ -14,11 +14,11 @@ save yourself a lot of time by asking first.
 
 How do you talk to us? There are several ways to get in touch:
 
-* [ocio-dev](https://groups.google.com/forum/#!forum/ocio-dev):
+* [ocio-dev](https://lists.aswf.io/g/ocio-dev):
 This is a development focused mail list with a deep history of technical
```

Comment:
This may confuse people given that the old ocio-dev archives have not been migrated over yet.  Perhaps at the end of the sentence add something like: 
[NOTE: The ocio-dev google group archives are in process of being migrated to the new aswf list.]

Reply:
Alternatively, we can wait to merge/deploy these changes until after the Google groups are moved.

Reply:
Either way is fine with me.